WoWsShipBuilder.Web/Components/ShipUpgradeSelector.razor

```diff
@@ -15,14 +15,14 @@
             {
                 <MudMenu AnchorOrigin="Origin.BottomCenter" TransformOrigin="Origin.TopCenter" PopoverClass="mt-1" ListClass="pa-0" >
                     <ActivatorContent>
-                        <MudTooltip Color="Color.Transparent" Placement="Placement.Right">
+                        <MudTooltip Color="Color.Transparent" Placement="Placement.Right" Delay="400">
```

Comment:
Remove or at least reduce delay. This is too slow for desktop

Reply:
Iirc that was set to 500 and you reduced it to 400 like all the other similar tooltips